wcfsetup/install/files/lib/http/middleware/HandleValinorMappingErrors.class.php

```diff
@@ -38,8 +39,11 @@ public function process(ServerRequestInterface $request, RequestHandlerInterface
             $errors = Messages::flattenFromNode($e->node())
                 ->formatWith(new PrependPath());
 
-            if ($this->prefersJson($request)) {
-                return new JsonResponse(
+            return match (Helper::getPreferredContentType($request, [
+                'text/html',
+                'application/json',
+            ])) {
```

Comment:
Not sure about the formatting here.

Reply:
I would split it into two separate statements. First assign `$acceptedType = Helper::…` and then match against the variable.